erpnext_com/www/docs/user/manual/en/asset/asset.md

```diff
@@ -62,22 +61,23 @@ An Asset Category needs to be assigned to that Item. Now the Asset can be create
     <img class="screenshot" alt="Asset Item" src="{{docs_base_url}}/assets/img/asset/asset-item.png">
 1. **Custodian**: The employee you're giving/assigning the asset to.
 1. **Department**: The department of the Custodian.
-3. **Is Existing Asset**: Check if you already own the asset, not purchased recently. The asset could also have been carried forward from the previous Fiscal Year. The existing assets which are partially/fully depreciated can also be created/maintained for future reference.
+1. **Is Existing Asset**: Check if you already own the asset, not purchased recently. The asset could also have been carried forward from the previous Fiscal Year. The existing assets which are partially/fully depreciated can also be created/maintained for future reference.
 1. **Opening Accumulated Depreciation**: The accumulated depreciation amount which has already been booked for an existing asset.
 1. **Number of Depreciations Booked**: Enter the number of already booked depreciations for an existing asset.
 1. **Current Value (After Depreciation)**: In case you are creating a record of an existing asset which has already been partially/fully depreciated, mention the current value of the asset. In case of a new asset, mention the purchase amount or leave it blank.
 1. **Next Depreciation Date**: Mention the next depreciation date, even if it is the first one. If the asset is an existing one and depreciation has already been completed, leave it blank.
 1. **Calculate Depreciation**: Enable this checkbox to calculate depreciation of Assets.
+1. **Allow Monthly Depreciation**: Enable this checkbox to distribute depreciation amount of an year into 12 months. Depreciation entries will be made every month on the date provided as Depreciation Start Date. For eg. If Available for use date is 22nd Nov 2019 and depreciation start date is 31st March 2020, first depreciation will be done on 30th Nov 2019 second on 31st Dec 2019 and so on. Amount will be distributed based on days left until next depreciation. 
 
 ## 4. Features
 
 ### 4.1 Depreciation
 
 * **Frequency of Depreciation (Months)**: The number of months between depreciations.
-* **Depreciation Method**: There are two options: Straight Line and Double Declining Balance.
+* **Total Number of Depreciations**: The total number of depreciations during the useful life. In case of existing assets which are partially depreciated, mention the number of pending depreciations. For example if you set frequency as 12 months and no. of depreciations as 3, 1 depreciation will be booked every 12 months for 3 years.
```

Comment:
```suggestion
* **Total Number of Depreciations**: The total number of depreciations during the useful life. In case of existing assets which are partially depreciated, mention the number of pending depreciations. For example if you set frequency as 12 months and no. of depreciations as 3, 1 depreciation will be booked every 12 months for 3 years.
```
```suggestion
* **Total Number of Depreciations**: The total number of depreciations during the useful life of the Asset. In case of existing assets which are partially depreciated, mention the number of pending depreciations. For example, if you set frequency as 12 months and no. of depreciations as 3, 1 depreciation will be booked every 12 months for 3 years.
```